scep/api/webhook/webhook.go

```diff
@@ -0,0 +1,65 @@
+package webhook
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"go.step.sm/linkedca"
+
+	"github.com/smallstep/certificates/authority/provisioner"
+	"github.com/smallstep/certificates/webhook"
+)
+
+// Controller controls webhook execution
+type Controller struct {
+	client   *http.Client
+	webhooks []*provisioner.Webhook
+}
+
+// New returns a new SCEP webhook Controller
+func New(webhooks []*provisioner.Webhook) (*Controller, error) {
+	return &Controller{
+		client:   http.DefaultClient,
```

Comment:
This should be configurable since hosted uses a custom transport that only allows connections to public IPs

Reply:
Fixed this in e8c1e8719d35aeedebd8ca19a407afc797a8f663.